configuration.go

```diff
@@ -46,6 +49,7 @@ type DefaultEntryPoints []string
 // String is the method to format the flag's value, part of the flag.Value interface.
 // The String method's output will be used in diagnostics.
 func (dep *DefaultEntryPoints) String() string {
+	//TODO :
```

Comment:
What is left to do ? :angel: (should be written here, or the `TODO` should go away :stuck_out_tongue_closed_eyes: )